doc/devel/dependencies.rst

```diff
@@ -0,0 +1,156 @@
+.. _dependencies:
+
+============
+Dependencies
+============
+
+Mandatory dependencies
```

Comment:
```suggestion
Mandatory
```

Reply:
Even though the whole page is about dependencies, IMHO it's slightly better to keep it explicit.